site/docs/v1/tech/messengers/message-templates.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: doc
+title: Message Templates
+description: Configuring a Message Template
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+Navigate to [breadcrumb]#Customizations -> Message Templates# to set up your templates.
+
+* <<Create a Message Template>>
+    ** <<Add Message Template>>
+    ** <<Template>>
+    ** <<Localization>>
+    ** <<Localization Details>>
+
+== Create a Message Template
+
+A message template allows you to customize how the message is sent to your audience. You can add custom styles and text using FreeMarker.
+
+A new template can be added by clicking on the icon:plus[type=fas, role=ui-button green] button.
+
+image::messengers/message-templates-home.png[Message Templates Home, width=1200]
+
+=== Add Message Template
+
+image::messengers/message-templates-add.png[Message Templates Add, width=1200]
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+A unique UUID is auto generated if left blank.
+
+[field]#Name# [required]#Required#::
+For display only. Name of the message template.
+
+[field]#Type# [optional]#Optional#::
+The type of message template. Options include: SMS, Kafka, and Generic.
+
+=== Template
+
+image::messengers/message-template-section.png[Message Templates, width=1200, role=top-cropped bottom-cropped]
+
+==== Form Fields
+
+[.api]
+[field]#Default Text# [required]#required#::
+This text is displayed in each message. The default text accepts replacement variables.
+Valid replacement variables are listed below.
+
+==== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.21.0#::
```

Comment:
Where did you get these variables from? 

Also, no need for the 'since' since this is all new in 1.26 as you mention above.

Reply:
@bguyza Was able to provide some of them in slack on the dev channel - apr 27th, at 12:26pm

For some of the explanations of the variables and fields, I stole information from email templates.  Would be good to confirm with @bguyza to make sure they are accurate.

```
So in short, we provide
tenant
application if not null
user if not null
email
mobilePhone
code
```

Reply:
if confirmed accurate I will add it to your message template section

Reply:
@jobannon 
Yes these are correct.  

You might optionally add a link to the references of `User API`, `Tenant API`, and `Application API` in the descriptions.